Models/ResourceModelMap.cpp

```diff
@@ -129,8 +129,9 @@ void ResourceModelMap::ResourceRemoved(TypeCase type, const QString& name,
     }
   }
 
-  // Remove an references to this resource
-  emit ResourceRenamed(ResTypeAsString(type), name, "");
```

Comment:
Do we have slots listening on this signal? If so, killing `ResourceRenamed` might break something (e.g. the EGM handler might want to know that this happened so it can issue a rename on the underlying files). It might be worth emitting both signals.

That said, this is probably underthought—that rename might fail (e.g. the resulting path might be too long or the name might conflict with another entity).